var/spack/repos/builtin/packages/siesta/package.py

```diff
@@ -35,10 +35,14 @@ class Siesta(Package):
 
     version('4.0.1', '5cb60ce068f2f6e84fa9184ffca94c08')
 
+    patch('configure.patch')
```

Comment:
Can you report this bug upstream?

Reply:
They actually dropped the configure script entirely in the current development version (https://bazaar.launchpad.net/~siesta-maint/siesta/trunk/revision/570#Src/configure):
> Removed all configure related stuff, as it has not been updated in several years it has been decided to remove until we can get it back in.

Reply:
Ok, then the patch should with `when='@4.0.1'`.